src/components/Home.tsx

```diff
@@ -0,0 +1,12 @@
+import app from 'config/firebase';
+
+const Home = () => {
```

Comment:
it can be exported here like 
export const Home = () => {

Reply:
fixed - add export